src/components/cesium_map/api/spatial.js

```diff
@@ -346,19 +314,18 @@ export class PointStreamPrimitiveCollection extends Cesium.PointPrimitiveCollect
           } else {
             locations[location] = 1;
           }
-          const p0 = Cesium.Cartesian3.fromDegrees(doc.x, doc.y, 10);
+          const p0 = Cesium.Cartesian3.fromDegrees(doc.x, doc.y, doc.z[0] + locations[location]);
```

Comment:
Right now, we directly use height from the endpoint plus the number of same location points.

Reply:
What does it do if there is no elevation returned?

Reply:
If there is no elevation returned, the point would be on the ground. I might need to add a function to check the `undefined` elevation returned.